cmd/root.go

```diff
@@ -39,3 +48,15 @@ func Execute() {
 		os.Exit(1)
 	}
 }
+
+func addTestFlags(cmds ...*cobra.Command) {
+	for _, cmd := range cmds {
+		cmd.Flags().StringVarP(&rlsName, "rls", "r", "", "name of the release you want to test")
+		cmd.Flags().StringVarP(&clientName, "client", "n", "", "name of the client you want to test")
+		cmd.Flags().StringVarP(&host, "host", "i", "127.0.0.1", "host used by seasonpackarr")
+		cmd.Flags().IntVarP(&port, "port", "p", 42069, "port used by seasonpackarr")
+		cmd.Flags().StringVarP(&apiKey, "api", "a", "", "api key used by seasonpackarr")
+
+		_ = cmd.MarkFlagRequired("rls")
+	}
+}
```

Comment:
You could add these as `PersistentFlags` on `testCmd`, that way all it's subcommands will have these.